File: aspnet/web-forms/overview/older-versions-security/roles/creating-and-managing-roles-cs/samples/sample13.cs
protected void RoleList_RowDeleting(object sender, GridViewDeleteEventArgs e)
{
	// Get the RoleNameLabel
	Label RoleNameLabel = RoleList.Rows[e.RowIndex].FindControl("RoleNameLabel") as Label;

	// Delete the role
	Roles.DeleteRole(RoleNameLabel.Text, false);

	// Rebind the data to the RoleList grid
	DisplayRolesInGrid();
}

File: aspnet/web-forms/overview/older-versions-security/roles/creating-and-managing-roles-vb/samples/sample13.vb
Protected Sub RoleList_RowDeleting(ByVal sender As Object,ByVal e As System.Web.UI.WebControls.GridViewDeleteEventArgs) Handles RoleList.RowDeleting
 ' Get the RoleNameLabel
 Dim RoleNameLabel As Label = CType(RoleList.Rows(e.RowIndex).FindControl("RoleNameLabel"),Label)

 ' Delete the role
 Roles.DeleteRole(RoleNameLabel.Text,False)

 ' Rebind the data to the RoleList grid
 DisplayRolesInGrid()
End Sub